packages/big-design-patterns/src/components/Header/Header.tsx

```diff
@@ -120,11 +120,12 @@ export interface HeaderProps {
   badge?: BadgeProps;
   description?: Description;
   icon?: ReactNode;
+  lozenge?: ReactNode;
```

Comment:
this would be open to abuse, client's could put just about another component in here and it would render

if you want to be truly restrictive, take a look at https://github.com/bigcommerce/big-design/blob/main/packages/big-design/src/components/Input/Input.tsx#L94-L97